servers/rendering/renderer_viewport.cpp

```diff
@@ -844,7 +844,7 @@ void RendererViewport::viewport_set_scaling_3d_scale(RID p_viewport, float p_sca
 }
 
 void RendererViewport::viewport_set_size(RID p_viewport, int p_width, int p_height) {
-	ERR_FAIL_COND(p_width < 0 && p_height < 0);
+	ERR_FAIL_COND(p_width < 0 || p_height < 0);
```

Comment:
Shouldn't it be `< 2` to match the Viewport code?

Reply:
I tried that but noticed that some places in the code base use 0 as size. This resulted in multiple editor-error messages. The current variant simply fixes the wrong `&&`. These other places should probably be investigated separately, but that would be quite far away from the linked issue.